pkg/cmd/recoveryapiserver/create.go

```diff
@@ -0,0 +1,127 @@
+package recoveryapiserver
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"path"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/client-go/tools/watch"
+
+	"github.com/openshift/cluster-kube-apiserver-operator/pkg/recovery"
+)
+
+type CreateOptions struct {
+	Options
+
+	KubeApiserverImage    string
+	StaticPodResourcesDir string
+	Timeout               time.Duration
+	Wait                  bool
+}
+
+func NewCreateCommand() *cobra.Command {
+	o := &CreateOptions{
+		Options:               NewDefaultOptions(),
+		KubeApiserverImage:    "", // TODO: set the public pullspec
+		StaticPodResourcesDir: "/etc/kubernetes/static-pod-resources",
+		Timeout:               5 * time.Minute,
+		Wait:                  true,
+	}
+
+	cmd := &cobra.Command{
+		Use: "create",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			err := o.Complete()
+			if err != nil {
+				return err
+			}
+
+			err = o.Validate()
+			if err != nil {
+				return err
+			}
+
+			err = o.Run()
+			if err != nil {
+				return err
+			}
+
+			return nil
+		},
+		SilenceErrors: true,
+		SilenceUsage:  true,
+	}
+
+	// TODO: add/prettify descriptions
+	AddFlags(cmd, &o.Options)
+
+	cmd.Flags().StringVar(&o.KubeApiserverImage, "kube-apiserver-image", o.KubeApiserverImage, "image reference for kubernetes apiserver")
+	cmd.Flags().StringVar(&o.StaticPodResourcesDir, "static-pod-resources", o.StaticPodResourcesDir, "path to store a directory containing static pod resources for recovery apiserver manifest")
+	cmd.Flags().DurationVar(&o.Timeout, "timeout", o.Timeout, "startup timeout, 0 means infinite")
+	cmd.Flags().BoolVar(&o.Wait, "wait", o.Wait, "wait for recovery apiserver to become ready")
```

Comment:
This command should always block until we give up on healthz or until the user cancels it.  Imagine it as the old `openshift start` logic that runs a server until the user terminates it.

Reply:
I don't really see this being an issue, we may have a script in the future, that already handles checking healthz or want to talk to apiserver before that. `openshift start` is a bit different because the process is actually the server. This just creates a config for another tool to run it and a distinct command to delete it

Reply:
and the default is to wait

Reply:
It's not about defaults.  it's about not having options we don't need or want.  These should not exist at all.

Reply:
> I don't really see this being an issue, we may have a script in the future, that already handles checking healthz or want to talk to apiserver before that. `openshift start` is a bit different because the process is actually the server. This just creates a config for another tool to run it and a distinct command to delete it

It should be a single command that runs a server until you cancel it and then the process terminates and cleans up.